fix/content-outdated (#27)
italia/design-linee-guida-docs: README.md

> # :warning: Questa versione delle linee guida è superata 

Reviewer: ```suggestion
# :warning: Queste Linee Guida sono superate, rimangono qui visionabili come memoria storica
```

> I riferimenti più attuali sono:

Reviewer: ```suggestion
I riferimenti attuali sono:
```

Reviewer 2: ```suggestion
I riferimenti normativi attualmente in vigore sono:
```

Reviewer 2: forse potremmo rinforzare ulteriormente @pedromaria88 che dici?

Author: @danielaiozzo il manuale non è normativo.

> - le **[Linee guida di design per i siti internet e i servizi digitali della PA](https://docs.italia.it/italia/design/lg-design-servizi-web/)** ([repository GitHub](https://github.com/italia/lg-design-servizi-web/)) emesse a norma CAD ex. Art 71;

Reviewer 2: ```suggestion
- le **[Linee guida di design per i siti internet e i servizi digitali della PA](https://docs.italia.it/italia/design/lg-design-servizi-web/)** ([repository GitHub](https://github.com/italia/lg-design-servizi-web/)), riferimento normativo in vigore, emesse a norma CAD ex. Art 71;
```

> - il **[Manuale operativo di design](https://docs.italia.it/italia/designers-italia/manuale-operativo-design-docs/)** ([repository GitHub](https://github.com/italia/manuale-operativo-design-docs)) e le altre risorse messe a disposizione come strumento a supporto della Pubblica Amministrazione e dei suoi fornitori dal progetto [Designers Italia](https://designers.italia.it).

Reviewer: ```suggestion
- il **[Manuale operativo di design](https://docs.italia.it/italia/designers-italia/manuale-operativo-design-docs/)** ([repository GitHub](https://github.com/italia/manuale-operativo-design-docs)) che raccoglie ed evolve gli aspetti operativi delle precedenti linee guida al presente link, e le altre risorse messe a disposizione come strumento a supporto della Pubblica Amministrazione e dei suoi fornitori dal progetto [Designers Italia](https://designers.italia.it).
```

> - il **[Manuale operativo di design](https://docs.italia.it/italia/designers-italia/manuale-operativo-design-docs/)** ([repository GitHub](https://github.com/italia/manuale-operativo-design-docs)) che raccoglie ed evolve gli aspetti operativi delle precedenti linee guida al presente link, e le altre risorse messe a disposizione come strumento a supporto della Pubblica Amministrazione e dei suoi fornitori dal progetto [Designers Italia](https://designers.italia.it).

Reviewer 2: ```suggestion
- il **[Manuale operativo di design](https://docs.italia.it/italia/designers-italia/manuale-operativo-design-docs/)** ([repository GitHub](https://github.com/italia/manuale-operativo-design-docs)) che raccoglie ed evolve gli aspetti operativi delle linee guida emesse a norma CAD e le altre risorse messe a disposizione come strumento a supporto della Pubblica Amministrazione e dei suoi fornitori dal progetto [Designers Italia](https://designers.italia.it).
```
